plugins/signals-backend/src/service/router.test.ts

```diff
@@ -43,6 +45,17 @@ const userInfo: jest.Mocked<UserInfoService> = {
   getUserInfo: jest.fn(),
 };
 
+const mockedAuthorize: jest.MockedFunction<PermissionsService['authorize']> =
+  jest.fn();
+const mockedPermissionQuery: jest.MockedFunction<
+  PermissionsService['authorizeConditional']
+> = jest.fn();
+
+const permissionsMock: PermissionsService = {
```

Comment:
mockServices here too - as a matter of fact the others above could get the same treatment too